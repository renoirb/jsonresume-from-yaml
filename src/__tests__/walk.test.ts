import { join, resolve } from "path";
import walker from "../walk";

const fixtureDir = resolve(join(__dirname, "__fixture__"));

describe("Walker", () => {
  it("Read directory file contents", async () => {
    const subject = await walker(fixtureDir);
    expect(subject).toMatchSnapshot();
  });
});
