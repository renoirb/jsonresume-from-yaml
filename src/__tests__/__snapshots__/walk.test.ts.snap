// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Walker Read directory file contents 1`] = `
Resume {
  "awards": Array [],
  "basics": Object {
    "email": "root@example.com",
    "label": "Software Developer",
    "location": Object {
      "city": "Montreal",
      "countryCode": "CA",
      "region": "Quebec",
    },
    "name": "John Doe",
    "picture": "http://facetheforce.today/luke/400",
    "profiles": Array [
      Object {
        "network": "Twitter",
        "url": "http://twitter.com/renoirb",
        "username": "renoirb",
      },
    ],
    "summary": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.
Nam lobortis vitae metus vitae fermentum. Nam nunc ex, egestas vel
Duis sit amet eleifend metus, eu laoreet felis.
",
    "website": "https://example.com",
  },
  "education": Array [],
  "interests": Array [],
  "languages": Array [],
  "publications": Array [],
  "references": Array [],
  "skills": Array [],
  "volunteer": Array [],
  "work": Array [],
}
`;
